docs/source/services/timelock_service/server_configuration.rst

```diff
@@ -0,0 +1,108 @@
+.. _timelock-server-configuration:
+
+Timelock Server Configuration
+=============================
+
+The Timelock Server configuration file is written in YAML and is located at ``var/conf/timelock.yml``.
+It has three main blocks: the ``clients`` block, ``cluster`` block and ``atomix`` block. We will discuss how each of
+these may be configured in turn, as well as additional configuration parameters.
+
+.. contents::
+   :local:
+
```

Comment:
I would like to see an example of a complete, valid server config. Is it possible to embed an actual file from our codebase?